lib/evm/functions.ex

```diff
@@ -100,15 +100,18 @@ defmodule EVM.Functions do
     metadata = Operation.metadata(instruction)
     dw = if metadata, do: Map.get(metadata, :input_count), else: nil
     aw = if metadata, do: Map.get(metadata, :output_count), else: nil
-    s0 = Stack.peek(machine_state.stack)
+    inputs = Operation.inputs(machine_state.stack, instruction)
```

Comment:
When we have too few inputs, where do we fail? Just want to make sure we don't end up with an top-level exception for bad code.

Reply:
I added  [this line](https://github.com/exthereum/evm/pull/17/files#diff-a13847d680d1e02cc1c208d323512d61R290) to return `[]` if we have too few inputs so this shouldn't cause any top level exceptions no matter what :)